### PR TITLE
fix(index.html): Fix "stryker shirt image" max-width, so it doesn't break the design on mobile browsers.

### PR DESCRIPTION
**Pull request by JellePetersHAN**
The image of the Stryker t-shirt currently breaks the design of the website on mobile browsers. To fix this the image should be (or rather has been) given a maximum width of _100%_.

### CODE REVIEW

File: stylesheets/stryker.css [hunk 1]
```diff
@@ -45,3 +45,8 @@ pre{
     padding: 0;
     text-align: left;
 }
+.stryker-shirt-image {
```
Comment (nicojs) [inline stylesheets/stryker.css]: This should be a `#` right?

**Comment by JellePetersHAN** [comment]
@nicojs Yeah, I mixed up class and id and noticed that a bit too late. I tried fixing it with a quick --amend,
 which I shouldn't have tried seeing as the resulting merge made things a bit worse.

A fix is on its way right now :persevere:

File: stylesheets/stryker.css (hunk 1 above)
Comment (JellePetersHAN) [inline stylesheets/stryker.css]: Yup, should be fixed now.

**Comment by nicojs** [comment]
I just realize I forgot to mention that we should also update the source files.

Could you update the src/index.pug file?

The HTML files are generated using [pugjs](https://pugjs.org). Run `npm start` locally to see the results of changes to the source files.

Sorry I forgot to mention earlier.

**Comment by JellePetersHAN** [comment]
@nicojs No problem, fixed it right away (using the usual workflow).

**Pull request merged**